plugins/octant/cmd/antrea-traceflow-plugin/main.go

```diff
@@ -0,0 +1,401 @@
+
+		protocol, err := request.Payload.String(protocolCol)
+		if err != nil {
+			log.Printf("unable to get dstPod at string : %s", err)
+		}
+		protocol = strings.ToUpper(protocol)
+
+		// Judge whether the name of trace flow is duplicated.
+		// If it is, then the user creates more than one traceflows in one second, which is not allowed.
+		tfName := srcPod + "-" + dstPod + "-" + time.Now().Format(TIME_FORMAT_YYYYMMDD_HHMMSS)
+		ctx := context.Background()
+		tfOld, err := a.client.OpsV1alpha1().Traceflows().Get(ctx, tfName, metav1.GetOptions{})
+		if err != nil {
+			log.Printf("unable to get traceflow \"%s\", detailed error: %s", tfName, err)
+		}
+		if tfOld.Name == tfName {
+			log.Printf("duplicate traceflow \"%s\": same source pod and destination pod in less than one second"+
+				": %+v. ", tfName, tfOld)
+			return nil
+		}
+
+		tf := &opsv1alpha1.Traceflow{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: tfName,
+			},
+			Spec: opsv1alpha1.TraceflowSpec{
+				Source: opsv1alpha1.Source{
+					Namespace: srcNamespace,
+					Pod:       srcPod,
+				},
+				Destination: opsv1alpha1.Destination{
+					Namespace: dstNamespace,
+					Pod:       dstPod,
+				},
+				Packet: opsv1alpha1.Packet{
+					IPHeader: opsv1alpha1.IPHeader{
+						Protocol: supportedProtocols[protocol],
+					},
+				},
+			},
+		}
+		var sport, dport int
+		if srcPort != "" {
+			sport, err = strconv.Atoi(srcPort)
+			if err != nil {
+				log.Printf("failed to get source port: %s", err)
+				return nil
+			}
+		}
+		if dstPort != "" {
+			dport, err = strconv.Atoi(dstPort)
+			if err != nil {
+				log.Printf("failed to get destination port: %s", err)
+				return nil
+			}
+		}
+		switch tf.Spec.Packet.IPHeader.Protocol {
+		case TCPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.TCP = &opsv1alpha1.TCPHeader{
+					SrcPort: int32(sport),
+					DstPort: int32(dport),
+				}
+			}
+		case UDPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.UDP = &opsv1alpha1.UDPHeader{
+					SrcPort: int32(sport),
+					DstPort: int32(dport),
+				}
+			}
+		case ICMPProtocol:
+			{
+				tf.Spec.Packet.TransportHeader.ICMP = &opsv1alpha1.ICMPEchoRequestHeader{
+					ID:       0,
+					Sequence: 0,
+				}
+			}
+		}
+		tf, err = a.client.OpsV1alpha1().Traceflows().Create(ctx, tf, metav1.CreateOptions{})
+		if err != nil {
+			log.Printf("failed to create traceflow CRD \"%s\", err: %s", tfName, err)
+			return nil
+		}
+		log.Printf("create traceflow CRD \"%s\" successfully, Traceflow Results: %+v", tfName, tf)
+		a.latestTfName = tf.Name
+		a.graph = graphviz.GenGraph(tf)
+		return nil
+	case showGraphAction:
+		name, err := request.Payload.String("name")
+		if err != nil {
+			return fmt.Errorf("unable to get name at string : %w", err)
+		}
+		// Invoke GenGraph to show
+		ctx := context.Background()
+		tf, err := a.client.OpsV1alpha1().Traceflows().Get(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			log.Printf("get traceflow CRD \"%s\" unsuccessfully, err: %s ", name, err)
```

Comment:
failed to get ....
Let's make log format to be consistent, like we can change all the "unable to", "unsuccessfully" to "failed to" .